src/parsers/sample-models.cpp

```diff
@@ -18,184 +18,153 @@
 
 #include "pinocchio/parsers/sample-models.hpp"
 
-#ifdef WITH_HPP_FCL
-#include <hpp/fcl/shape/geometric_shapes.h>
-#endif
-
 namespace se3
 {
   namespace buildModels
   {
+    
+    template<typename JointModel>
+    void addJointAndBody(Model & model, const JointModelBase<JointModel> & joint, const std::string & parent_name, const std::string & name)
```

Comment:
I think it is not necessary to add such things inside the Model class. I added it just as a shortcut, nothing more.